app/controllers/cities_controller.py

```diff
@@ -3,7 +3,8 @@
 from flask import jsonify
 
 from app.exceptions.state_exc import StateNotFoundError
-from app.services.cities_services import get_cities_from_state, get_states_and_cities
+from app.services.cities_services import (get_cities_from_state,
```

Comment:
the imports are in a different pattern than the rest of the code, probably forgot to enter `black .`.

Reply:
I agree!